src/pages/receipts/packageReceipt.js

```diff
@@ -723,7 +723,43 @@ const controlCollectionCardField = (dropdownSelection) => {
     document.getElementById('collectionCheckBox').removeAttribute("disabled")
     enableCollectionCardFields() 
   }
-} 
+}
+export const validatePackageInformation = () => {
+  
+  const selectPackageConditionsList = document.getElementById('packageCondition').getAttribute('data-selected');
+  const parseSelectPackageConditionsList = JSON.parse(selectPackageConditionsList);
+  const scannedBarcode = document.getElementById("scannedBarcode").value;
+  const dateReceived = document.getElementById("dateReceived").value;
+  const collectionId = document.getElementById("collectionId").value;
+  const dateCollectionCard = document.getElementById("dateCollectionCard").value;
+  const timeCollectionCard = document.getElementById("timeCollectionCard");
+
+  return (parseSelectPackageConditionsList.length !== 0) &&
+    !!scannedBarcode && !!dateReceived && !!collectionId && !!dateCollectionCard && !!timeCollectionCard;
+};
+
+export const displayInvalidPackageInformationModal = (modalHeaderEl,modalBodyEl) => {
```

Comment:
This new function may not be needed. One of existing functions (`triggerErrorModal`, `showNotifications` `showTimedNotifications`) may work for you.

Reply:
As I recall, I initially tried these functions and they did not succeed. I needed to specifically utilize the same modal header and body elements as in the success case (displaySelectedPackageConditionListModal) to avoid issues with the submit button pointing at the modal via `data-toggle="modal" data-target="#modalShowMoreData"`. A fuller refactor could achieve this functionality in an alternate fashion, but right now this is the most straightforward approach.

Reply:
Agreed. Not sure when or whether the refactor will happen.